generators/php/sdk/src/core/RawClient.ts

```diff
@@ -19,6 +19,8 @@ export declare namespace RawClient {
         queryBagReference?: string;
         /** The reference to the request body */
         bodyReference?: php.CodeBlock;
+        /** The reference to the request body class */
+        classReference: php.ClassReference;
```

Comment:
Let's call this `requestTypeClassReference`, otherwise it's a little too easy to mix up with the others (e.g. `clientReference` above).